### PR TITLE
updated modal

### PR DESCRIPTION
**Pull request by A1pha0ps**
- made the modal somewhat more modularized
- changed it so it doesn't bring you back up to page when you close it

### CODE REVIEW

File: src/routes/+page.svelte [hunk 1]
```diff
@@ -188,4 +192,8 @@
 			opacity: 1;
 		}
 	}
+
+	.btn:focus {
+		outline: none !important;
+	}
```
Comment (jay-joo-code) [inline src/routes/+page.svelte:198]: it's usually good to have some sort of indicator of focused elements ... 
was the outline too ugly?

**Review by jay-joo-code — CHANGES_REQUESTED** [review]
overall amazing!!
just 1 nit, but should be able to merge it in soon

File: src/routes/+page.svelte (hunk 1 above)
Comment (A1pha0ps) [reply src/routes/+page.svelte:198]: When I opened the modal, there were 2 blue borders around random elements by default. It kinda bothered me so I'm not sure what to do... (It's a blue border around the insta button and the bottom left corner of the yellow div)

Comment (jay-joo-code) [reply src/routes/+page.svelte:198]: oh maybe it's autofocusing random elements in the modal on show 
do you want to try moving this css style to the Modal component you made if the issue was specifically for the modal?
it's generally not great to remove all focus indicators for a whole page

Comment (jay-joo-code) [reply src/routes/+page.svelte:198]: you can also use `outline-none` tailwind class to remove outlines!
https://tailwindcss.com/docs/outline-style

Comment (A1pha0ps) [reply src/routes/+page.svelte]: Fixed it! I thought it was the buttons but it was the <a> tags that was causing it

File: src/lib/components/glue/Modal.svelte [hunk 2]
```diff
@@ -0,0 +1,33 @@
+<script lang="ts">
+	/**
+	 * @type {boolean}
+	 */
+	export let showModal: boolean;
```
Comment (jay-joo-code) [inline src/lib/components/glue/Modal.svelte:5]: we don't really need doc strings because typescript `: boolean` already denotes type

**Review by jay-joo-code — CHANGES_REQUESTED** [review]

File: src/lib/components/glue/Modal.svelte (hunk 2 above)
Comment (A1pha0ps) [reply src/lib/components/glue/Modal.svelte]: fixed

**Review by jay-joo-code — APPROVED** [review]
awesome stuff

**Pull request merged**